src/code/z_horse.c

```diff
@@ -1,7 +1,15 @@
 #include "global.h"
 #include "vt.h"
 
-s32 func_8006CFC0(s32 scene) {
+#define HORSETYPE_1 1
+#define HORSETYPE_2 2
+#define HORSETYPE_5 5
+#define HORSETYPE_6 6
+#define HORSETYPE_7 7
+#define HORSETYPE_8 8
+#define HORSETYPE_9 9
+
+s32 Horse_CheckSceneValid(s32 scene) {
```

Comment:
I think this function name isn't specific enough, from this name I expected the function to be used as "can a horse spawn in this scene" but it doesn't look like that's it

It looks like the scenes where you can leave Epona and find her at the same spot any time maybe? not sure

Reply:
I will look into it and create a new commit as soon as I know more.
According to z_horse.c:256, this function was once called Horse_Set_Check(). 
That name could be better imho. What do you think?
```c
//Line 255...
void func_8006DC68(GlobalContext* globalCtx, Player* player) {
    if (LINK_IS_ADULT) {
        if (!Horse_CheckSceneValid(gSaveContext.horseData.scene)) {
            osSyncPrintf(VT_COL(RED, WHITE));
            // "Horse_Set_Check():%d set spot is no good."
            osSyncPrintf("Horse_Set_Check():%d セットスポットまずいです。\n", gSaveContext.horseData.scene);
            osSyncPrintf(VT_RST);
            func_8006D074(globalCtx);
        }
//Line 265...
```

Reply:
We very rarely name things based on names from debug strings, since those names usually weren't very good (it's pretty safe to say many devs didn't master english)

In this case I don't think `Horse_Set_Check` is any good, as the name probably should mention being a scene check (which `Horse_CheckSceneValid` does) and also the check's purpose as I said above

Reply:
I thought a little more, how about `Horse_CanSpawnInScene`?

Reply:
I don't think that's accurate, there are plenty `Actor_Spawn` calls to spawn Epona that get called without this function playing a role